### PR TITLE
Image denoising

### PR DESCRIPTION
**Pull request by wolski0420**

### CODE REVIEW

File: denoising.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    Returns
+    -------
+    numpy.ndarray
+        denoised image
+
+    """
+
+    # median filtering to remove useless points or small blurs
+    med_img = median(image)
```
Comment (j-adamczyk) [inline denoising.py:26]: Can potentially clash with median from statistics, better call `skimage.filters.median`, meaning is also clearer

File: denoising.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    threshold = threshold_otsu(gray_img)
+    bin_img = gray_img > threshold
+
+    # denoising process
+    result_img = denoise(bin_img)
+
+    # saving image to file with scikit image function
+    imsave(f"{filename.split('.')[0]}_res_col070.jpg", img_as_ubyte(result_img))
+
+
```
Comment (j-adamczyk) [inline denoising.py:65]: Only 1 empty line is needed

File: denoising.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    # binarisation process
+    threshold = threshold_otsu(gray_img)
+    bin_img = gray_img > threshold
+
+    # denoising process
+    result_img = denoise(bin_img)
+
+    # saving image to file with scikit image function
+    imsave(f"{filename.split('.')[0]}_res_col070.jpg", img_as_ubyte(result_img))
```
Comment (j-adamczyk) [inline denoising.py:63]: Remove for merging purposes

**Comment by wolski0420** [comment]
I made changes as you suggested

**Review by konrrad — APPROVED** [review]

**Pull request merged**